lib/rucio/common/pcache.py

```diff
@@ -1045,15 +1044,16 @@ def daemonize(self) -> None:
         try:
             os.setsid()
         except OSError:
-            pass
+            self.log(DEBUG, "Failed to make a new session for daemon")
         try:
             os.chdir("/")
         except OSError:
-            pass
+            self.log(DEBUG, "Failed to change dir to '/'")
         try:
             os.umask(0)
         except OSError:
-            pass
+            self.log(DEBUG, "Failed to unmask dir")
```

Comment:
```suggestion
            self.log(DEBUG, "Failed to umask dir")
```

https://docs.python.org/3/library/os.html#os.umask